src/spanish.js

```diff
@@ -0,0 +1,16 @@
+class Spanish {
+  conjugate(word,info){
+
```

Comment:
@trifrog Nice work!
Could you also add a switch with cases for the tenses? You can just add one case Present tense for now.